Bottom: bc795d110afbf9301da2ff631ec0a3c1c331f362
Top:    53e04c78a4ce5a1a7cad95c1b58f0eb08ee19b01
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 19:35:50 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 7980090..97f92a2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
